packages/expo/ios/AppDelegates/ExpoReactNativeFactory.swift

```diff
@@ -1,16 +1,20 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import React_RCTAppDelegate
+
 /**
  Implements a subclass of EXReactNativeFactory. By having both an objective-c and a swift override we can utilise
```

Comment:
nit: This comment is not valid anymore after you simplified the way these factories are setup.

Reply:
good catch! it's removed. thanks!